core/builtin.rbs

```diff
@@ -266,7 +266,12 @@ type real = Integer | Float | Rational
 #
 # A lot of builtin functions accept either a Symbol, a String, or something which has `.to_str`
 # defined.
+# 
 type interned = Symbol | string
+# Previous incarnations of `interned` for backward-compatibility (see #1499)
+%a{steep:deprecated} type Object::name = interned
+%a{steep:deprecated} type Module::id = interned
+%a{steep:deprecated} type Net::HTTPHeader::key = interned
```

Comment:
Found this annotation in [`schema/annotation.json` L3](https://github.com/ruby/rbs/blob/fce99b9700f5b9c9749e7a8c65852dc4b6b40cc7/schema/annotation.json#L3) but otherwise no deprecation annotation documented in either RBS or Steep. Not sure if it’s appropriate.